### DISCUSSION TITLE
classes/class_scenemultiplayer

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#134] [User-contributed notes]
# classes/class_scenemultiplayer

Inherits: MultiplayerAPI< RefCounted< Object High-level multiplayer API implementation. Description: This class is the default implementation of MultiplayerAPI, used to provide multiplayer function...

https://docs.godotengine.org/en/stable/classes/class_scenemultiplayer.html

**Comment by SamTheBlow** [comment]
The `auth_timeout` property is an amount of time **measured in seconds**.

**Reply by Calinou** [reply]
Pull request opened:

- https://github.com/godotengine/godot/pull/97045

**Comment by FeralBytes** [comment]
If you call disconnect_peer(peer_id) does not trigger the inherited signal of MultiplayerAPI.peer_disconnected for the server, but it will trigger this signal for all clients.
